kapitan/refs/base.py

```diff
@@ -99,6 +91,54 @@ def from_params(cls, data, ref_params):
             # data already bytes encoded
             return cls(data, **ref_params.kwargs)
 
+    def dump(self):
+        """
+        Returns dict with keys/values to be serialised.
+        """
+        return {"data": self.data.decode(), "encoding": self.encoding,
+                "type": self.type_name}
+
+
```

Comment:
Would it make sense to split these classes to separate files? They're more implementations of the base classes than actual "base.py"(?).

Reply:
makes sense, will work on it